src/common/createplay/CreatePlay.jsx

```diff
@@ -121,7 +121,7 @@ const CreatePlay = () => {
     if (refreshToken) {
       console.log(refreshToken);
     } else {
-      window.location = NHOST.AUTH_URL;
+      window.location = NHOST.AUTH_URL('http://localhost:3000/plays/create');
```

Comment:
Why?

Reply:
this is bcz we need redirect user based on auth which is different in two different usecase. thats y we are passing redirect url dynamically. auth while liking a play (when user wants to like a play) we always want to redirect user to that specefic play after auth and slo for create play (after auth) we want to redirect user into /plays/create route.

Reply:
I guess both should work with `window.location.href`. We can still maintain uniformity